net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,181 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=http://www.yo2loj.ro/hamprojects
+PKG_MD5SUM:=5429401ab6f8f793448bef11029d7ead
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)    
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	DEPENDS:=+kmod-ipip +ip-full
+	TITLE:=Routing daemon for the AMPR network
+	URL:=http://www.yo2loj.ro/hamprojects
+	MAINTAINER:=Dan Srebnick <k2ie@k2ie.net>
+endef
+
+define Package/$(PKG_NAME)/description
+	Routing daemon written in C similar to Hessu's rip44d including optional resending of RIPv2 broadcasts for router injection.
+endef
+
+CONFIGURE_VARS+= \
+	CC="$(TOOLCHAIN_DIR)/bin/$(TARGET_CC)"
+	COPT="$(TARGET_COPT)"
+
+define Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/$(PKG_NAME)-init $(1)/etc/init.d/$(PKG_NAME)
+endef
+
+define Package/$(PKG_NAME)/postinst
+#!/bin/sh
+if [ -z "$${IPKG_INSTROOT}" ]; then
+   echo "Installing with values:
+
+  amprhost=$$amprhost
+  amprmask=$$amprmask
+  amprnet=$$amprnet
+"
+   echo "Creating /etc/ampr-ripd.conf..."
+   echo "# Set to your AMPR allocation
+# eg. 44.128.0.32/255.255.255.240
+TUNNET=$$amprnet/$$amprmask" > /etc/ampr-ripd.conf
+   
+   echo "Installing routing rules..."
+   r=`uci add network rule`
```

Comment:
Backtick notation is considered legacy: https://www.shellcheck.net/wiki/SC2006

Reply:
That is correct.  For unknown reasons, the $(...) notation totally breaks the ability to add rules and insert the related configuration items.

Reply:
You probably just needed to escape the `$` like `$$(...)`. This is not going to be an issue if you use anything other than the `postinst` hook.